service_management/azure/test/integration/service_bus/rules_test.rb

```diff
@@ -14,84 +14,92 @@
 #--------------------------------------------------------------------------
 require "integration/test_helper"
 
-describe "ServiceBus Rules" do
+describe Azure::ServiceBus::ServiceBusService do
   subject { Azure::ServiceBus::ServiceBusService.new }
-  after { ServiceBusTopicNameHelper.clean }
-  let(:topic){ ServiceBusTopicNameHelper.name }
+  let(:topic){ 'test-topic' }
   let(:subscription){ "mySubscription" }
   let(:rule){ "myRule" }
 
-  before {
-    subject.create_topic topic
-    subject.create_subscription topic, subscription
-  }
-
-  it "should be able to create a new rule" do
-    result = subject.create_rule topic, subscription, rule
-    result.must_be :kind_of?, Azure::ServiceBus::Rule
-    result.filter.must_be_kind_of Azure::ServiceBus::TrueFilter
-    result.filter.sql_expression.must_equal "1=1"
-    result.filter.compatibility_level.must_equal 20
-  end
+  describe "Service bus rules" do
+    before do
+      VCR.insert_cassette "service_bus/#{name}"
+      subject.create_topic topic
+      subject.create_subscription topic, subscription
+    end
 
-  it "should be able to create a new rule object" do
-    ruleObject = Azure::ServiceBus::Rule.new "my_other_rule"
-    ruleObject.subscription = subscription
-    ruleObject.topic = topic
+    after do
+      subject.delete_topic topic
+      VCR.eject_cassette
+    end
 
-    result = subject.create_rule ruleObject
-    result.must_be :kind_of?, Azure::ServiceBus::Rule
-    result.filter.must_be_kind_of Azure::ServiceBus::TrueFilter
-    result.filter.sql_expression.must_equal "1=1"
-    result.filter.compatibility_level.must_equal 20
+    it "should create a new rule" do
+      result = subject.create_rule topic, subscription, rule
 
-    subject.delete_rule result
```

Comment:
actually deleting topic will chain delete the rules & subscriptions, so we do not explicitly delete it.